python/makeALPACAEvents.py

```diff
@@ -139,6 +144,59 @@ def runEvents(mres,gax,nev,Lmin,Lmax,startZ,endZ,SmearBeam):
     print('Events are recording into a ntuple.')
     inputFile = ntupleWrite(ctau,fn,Lmin,Lmax,startZ,endZ,SmearBeam,mres,gax,old)
     os.chdir(old)
+    os.system("rm "+wdir+"/input_"+str(mres)+"_"+str(gax)+".DAT")
+    os.system("rm "+wdir+"/evrecs/evrec_"+str(mres)+"_"+str(gax)+".dat")
+    os.system("rm "+wdir+"/outputs/output_"+str(mres)+"_"+str(gax)+".dat")
```

Comment:
It's better to use `os.remove`.

Reply:
is there a way to have this intermediate file in memory? How big are these files usually in MB?

Reply:
Don't know about the file size, but there are ways to have ROOT and non-ROOT files in memory only (or as temporary files).

Reply:
I used os.remove.
Thank you for your tip.
Done.

Reply:
> is there a way to have this intermediate file in memory? How big are these files usually in MB?

Honestly, I did not know. I will have a look and report here.
Since ALPACA needs these files inside bin dir, I did not want to mess the sw dir and made this deletion.